tests/test_parser.py

```diff
@@ -4,6 +4,15 @@
     curve_order, negative_G1
 
 
+def test_block_number():
+    block_number_code = """
+def block_number() -> num:
+    return block.number
```

Comment:
What does this have to do with exponents?

Reply:
Same coveralls situation here.